lib/cli.js

```diff
@@ -79,7 +84,7 @@ async function check(argv) {
         )} flag to ignore these errors and force the record to be rewritten.`
       );
       process.exit(1);
-    } else if (results.filter(({ type }) => type !== 'info').length === 0) {
+    } else if (results.filter(({ type }) => type !== "info").length === 0) {
```

Comment:
This change was made by prettier every time I saved. I can revert it though, if you want.

```suggestion
    } else if (results.filter(({ type }) => type !== 'info').length === 0) {
```

Reply:
you can leave it in